src/ale/4C_ale_ale2.cpp

```diff
@@ -70,7 +70,7 @@ void Discret::Elements::Ale2Type::nodal_block_information(
 /*----------------------------------------------------------------------------*/
 /*----------------------------------------------------------------------------*/
 Core::LinAlg::SerialDenseMatrix Discret::Elements::Ale2Type::compute_null_space(
-    Core::Nodes::Node& node, const double* x0, const int numdof, const int dimnsp)
+    Core::Nodes::Node& node, std::array<double, 3> x0, const int numdof)
```

Comment:
Please use std::span instead of std::array.
```suggestion
    Core::Nodes::Node& node, std::span<double, 3> x0, const int numdof)
```

And is the 3 correct for ale2? It might need to be coordinated with #1283